src/App.jsx

```diff
@@ -14,70 +15,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {
+          selectedGood === ''
+            ? (
+              'No goods selected'
+            )
+            : (
+              <>
+                { `${selectedGood} is selected` }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+                <button
+                  onClick={() => {
+                    setGood('');
+                  }}
```

Comment:
make function which will take good and retur arrow function and use in all places
```
const makeSetGood = (good) => () => setGood(good);
```
```suggestion
                  onClick={makeSetGood('')}
```

Reply:
const makeSetGood = (good) => () => setGood(good); 
tell me about this syntax pls, thx

Reply:
why we need to use syntax like that